microsoft-edge/progressive-web-apps-chromium/landing/index.yml

```diff
@@ -0,0 +1,150 @@
+title: Progressive Web Apps (PWAs) documentation
+summary: Develop a web app that has a progressive UX and can run on all devices, from one codebase.  Provide a native-like UX, or run in the browser.  # < 160 chars; cursor 169
+
+metadata:
+  # title: default to root title above. # Required; page title displayed in search results. Include the brand. < 60 chars.
+  # description: default to root title above. # Required; article description that is displayed in search results. < 160 chars.
+  ms.service: microsoft-edge
+  ms.subservice: pwa
+  ms.topic: landing-page
+  # ms.collection: collection # Optional; Remove if no collection is used.
+  author: MSEdgeTeam
+  ms.author: msedgedevrel
+  ms.date: 11/26/2024
+
+# =============================================================================
+# Valid article types (sections of a card)
+
+      # - linkListType: architecture
+      # - linkListType: concept
+      # - linkListType: deploy
+      # - linkListType: download
+      # - linkListType: get-started
+      # - linkListType: how-to-guide
+      # - linkListType: learn
+      # - linkListType: overview
+      # - linkListType: quickstart
+      # - linkListType: reference
+      # - linkListType: sample
+      # - linkListType: tutorial
+      # - linkListType: video
+      # - linkListType: whats-new
+
+# =============================================================================
+landingContent:
+
+# =============================================================================
+# Card r1c1
+  - title: Overview and getting started
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: Overview of Progressive Web Apps (PWAs)
+            url: ../index.md
+
+          - text: Use PWAs in Microsoft Edge
+            url: ../ux.md
+
+      - linkListType: get-started
+        links:
+          - text: Get started with PWAs
+            url: ../how-to/index.md
+
+      - linkListType: whats-new
+        links:
+          - text: What's New in PWAs
+            url: ../whats-new/pwa.md
+
+# =============================================================================
+# Card r1c2
+  - title: Integrate a PWA into Windows
+    linkLists:
+      - linkListType: how-to-guide
+        links:
+          - text: Display content in the title bar area using Window Controls Overlay
+            url: ../how-to/window-controls-overlay.md
+
+          - text: Share content with other apps
+            url: ../how-to/share.md
+
+          - text: Define app shortcuts (long-press or right-click menus)
+            url: ../how-to/shortcuts.md
+
+          - text: Display a PWA widget in the Windows Widgets Board
+            url: ../how-to/widgets.md
+
+          - text: Build a PWA for the sidebar in Microsoft Edge
+            url: ../how-to/sidebar.md
+
+# =============================================================================
+# Card r1c3
+  - title: Publish PWAs with the Microsoft Store
```

Comment:
name of Store card in pwa landing page
alts:
Publish PWAs with the Microsoft Store <-- simple, emph store, quickly clear, differentiates other buckets
Publish and integrate PWAs with the Microsoft Store
Publish and integrate with the Microsoft Store - missing subject 'pwa'; 'integrate' adds unclarity; keyword 'store' delayed/diluted
Publish a PWA with the Microsoft Store
Microsoft Store publishing and integration
Publish and integrate with the Microsoft Store
Microsoft Store: Publishing and integrating a PWA

toc equiv PR [Comment](https://github.com/MicrosoftDocs/edge-developer/pull/3320/files/150ef4fc089397de10e9341ef0eb7ec457beed0f#r1877438481)

Reply:
I agree that "integrate" is unclear, but if we only say "publish", then I have to look at each article's title to understand that the content of the bucket is not limited to publishing into the store.

How about "Distribute a PWA with the Microsoft Store"?
This would align with the intro in https://learn.microsoft.com/en-us/windows/apps/publish/ and I feel like "distribute" is a more general term that ecompasses both submitting the app to the store, and leveraging other store distribution mechanisms (which would include in-app purchases).

Reply:
changed all 2x: 
toc
pwa landing